pandapipes/create.py

```diff
@@ -911,18 +916,19 @@ def create_circ_pump_const_mass_flow(net, from_junction, to_junction, p_bar, mdo
 def create_fluid_from_lib(net, name, overwrite=True):
     """
     Creates a fluid from library (if there is an entry) and sets net["fluid"] to this value.
-    Currently existing fluids in the library are: "water", "natural_gas", "air".
+    Currently existing fluids in the library are: "hgas", "lgas", "water", "air", "carbondioxid",
+    "ethane", "hydrogen", "methane", "nitrogen" and "oxygen"
```

Comment:
call_lib does not work with  "carbondioxid", "ethane", "hydrogen", "methane", "nitrogen" and "oxygen"
```suggestion
    Currently existing fluids in the library are: "hgas", "lgas", "water", "air".
```